src/ci/bin/testCentaurAws.sh

```diff
@@ -30,12 +30,8 @@ cromwell::build::run_centaur \
     -e non_root_default_user \
     -e draft3_glob_access \
     -e bad_output_task \
-    -e cachewithinwf \
```

Comment:
TOL I never really understood why these files existed rather than listing the allowable (or excluded) backends in the `.test` files (which is also possible AFAIK?)

Reply:
🤷‍♂️ This is what I was shown to edit in a previous aws/centaur PR. TBH I like this better as it puts it easily visible in one place - e.g. if I ask the question "what's the set of known to be failing centaur tests", well now I know.

Reply:
> rather than listing the allowable (or excluded) backends

AFAIK, there is not a way using only the `.test` files to say "run this test on PapiV2, Local, Slurm, but NOT AWS, PapiV1, Alibaba, TES". Thus we ended up with each `testCentaur*.sh` either opting-in or opting-out a set of tests.